### ISSUE TITLE
Setting Tags with API Command

### ISSUE THREAD

**Issue by nbamrah** [ConfigHubPub/ConfigHubPlatform#83]
This is not an issue, more of a request. I would like to be able to set tags using an api command. It appears that there is no way to do this, other than the UI.

**Comment by fbunta** [comment]
I agree, without being able to set a tag in a push request the functionality of tags as a whole is limited.

**Comment by edoarsla** [comment]
oh.  that's a good point.  should be easy enough to add.
